### DISCUSSION TITLE
Export to curl from Addon

### DISCUSSION THREAD

**Discussion by bw9ubwo** [mitmproxy/mitmproxy#7211] [Q&A]
Hi. As the title suggests. I try to access the curl export from an addon. Is this possible?

**Comment by mhils** [comment]
My recommendation would be to just import what you need from [`mitmproxy.addons.export`](https://github.com/mitmproxy/mitmproxy/blob/main/mitmproxy/addons/export.py). This is not part of our public documented API and may change at any time without notice, but practically speaking you should be relatively fine. :)
